zeppelin-zengine/src/main/java/com/nflabs/zeppelin/notebook/Note.java

```diff
@@ -231,6 +232,9 @@ public void run(String paragraphId) {
     p.setNoteReplLoader(replLoader);
     p.setListener(jobListenerFactory.getParagraphJobListener(this));
     Interpreter intp = replLoader.get(p.getRequiredReplName());
+    if (intp == null) {
```

Comment:
@Leemoonsoo  thank you for the review.

I have made the following changes:
1. for NPEs getMessage() is null, hence no error is shown on the GUI. I have fixed this in InterpreterUtils.getMostRelevantMessage
2. added a test for this specific case to throw an InterpreterException instead of NPE